### PR TITLE
DLSV2-520 Removed Confirmed/Unconfirmed statuses from ManageSupervisors view

### PR DESCRIPTION
**Pull request by daniel-manta-hee**
This change was requested in comment https://hee-dls.atlassian.net/browse/DLSV2-520?focusedCommentId=10091 on the below Jira issue.

Also removing redundant code relative to `Confirmed` column on table `SupervisorDelegates`.

### JIRA link
https://hee-dls.atlassian.net/browse/DLSV2-520

### Description
Removed statuses column (Confirmed/Unconfirmed) and action links: Confirm/Reject from ManageSupervisors view.

### Screenshots
image

### Developer checks

**Send invite**
- Go to Supervise/My staff list/Add a member of staff
- Enter an email
- Submit
- Checked that invitation email is sent

**Breakpoints:** 
- DigitalLearningSolutions.Data\Services\FrameworkNotificationService.cs line 201
- DigitalLearningSolutions.Data\Services\EmailService.cs line 52 (skip Authenticate and SendSingleEmailFromClient)
- From `builder.TextBody` copy url link.

Either logout or use below breakpoint and skip `if (User.Identity.IsAuthenticated)`:
- C:\Repos\DLSV2\DigitalLearningSolutions.Web\Controllers\Register\RegisterController.cs line 68

**Complete registration**
- Visit the link: replace server by your local host and visit the link
- Enter personal information (remember email)
- Enter Learner information
- Create password (remember password)
- Submit
- Delegate registration complete

image

**Check that you can log in as the delegate**
- Go to My Account
- Log out
- Log in

**Check that you have access to launch course**

image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ManageSupervisors.cshtml [hunk 1]
```diff
@@ -48,31 +45,17 @@
...
-              }
-            </td>
             <td class="nhsuk-table__cell">
               <span class="nhsuk-table-responsive__heading">Actions </span>
               @if (supervisor.AddedByDelegate)
               {
-                <a asp-action="RemoveSupervisor" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-route-supervisorDelegateId="@supervisor.ID">Remove</a>
+                <a asp-action="RemoveSupervisor" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-route-supervisorDelegateId="@supervisor.ID">Remove</a>
+                @Html.Raw("&nbsp;")
                 @if (supervisor.Confirmed == null && supervisor.NotificationSent.ToShortDateString() != DateTime.UtcNow.ToShortDateString())
```
Comment (kevwhitt-hee) [inline DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ManageSupervisors.cshtml:54]: We shouldn't need the send reminder functionality either.

**Review by kevwhitt-hee — CHANGES_REQUESTED** [review]
Can we also remove the redundant send reminder link - see code comment.

File: DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ManageSupervisors.cshtml (hunk 1 above)
Comment (daniel-manta-hee) [reply DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ManageSupervisors.cshtml]: Done. Added new commit for removing "send reminder" link in Manage Supervisors.

**Review by kevwhitt-hee — APPROVED** [review]
Good.

**Pull request merged**